aviary/interface/methods_for_level2.py

```diff
@@ -331,7 +334,7 @@ def load_inputs(self, aviary_inputs, phase_info=None, engine_builder=None):
 
         ## PROCESSING ##
         # set up core subsystems
-        if mission_method is HEIGHT_ENERGY:
+        if mission_method in (HEIGHT_ENERGY, SOLVED_2DOF):
             everything_else_origin = FLOPS
         elif mission_method in (TWO_DEGREES_OF_FREEDOM, SOLVED):
```

Comment:
Agreed. Let's take this into account as I focus on revamping things; this is really because there's still the notion of FLOPS and GASP in places where it doesn't need to be.